File: Calculator/app/src/main/java/com/example/calculator/MainActivity.java
package com.example.calculator;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    EditText etNum1,etNum2;
    Button btPlus,btMinus,btMultiply,btDevide;
    TextView tvAns;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        etNum1=findViewById(R.id.etNum1);
        etNum2=findViewById(R.id.etNum2);

        btPlus=findViewById(R.id.btAdd);
        btMinus=findViewById(R.id.btSubtract);
        btMultiply=findViewById(R.id.btMultiply);
        btDevide=findViewById(R.id.btDevide);

        tvAns=findViewById(R.id.tvAns);

        btPlus.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int num1,num2,ans;
                num1 = Integer.parseInt(etNum1.getText().toString());
                num2 = Integer.parseInt(etNum2.getText().toString());
                ans = num1 + num2;
                tvAns.setText("Ans is = "+ans);
            }
        });
        btMinus.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int num1,num2,ans;
                num1 = Integer.parseInt(etNum1.getText().toString());
                num2 = Integer.parseInt(etNum2.getText().toString());
                ans = num1 - num2;
                tvAns.setText("Ans is = "+ans);
            }
        });
        btMultiply.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int num1,num2,ans;
                num1 = Integer.parseInt(etNum1.getText().toString());
                num2 = Integer.parseInt(etNum2.getText().toString());
                ans = num1 * num2;
                tvAns.setText("Ans is = "+ans);
            }
        });
        btDevide.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int num1,num2,ans;
                num1 = Integer.parseInt(etNum1.getText().toString());
                num2 = Integer.parseInt(etNum2.getText().toString());
                ans = num1 / num2;
                tvAns.setText("Ans is = "+ans);
            }
        });
    }
}

File: RadioButton/app/src/main/java/com/example/radiobutton/MainActivity.kt
package com.example.radiobutton

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.CheckBox
import android.widget.RadioButton
import android.widget.RadioGroup
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val rgGroup = findViewById<RadioGroup>(R.id.rgGroup)
        val rbSalad = findViewById<RadioButton>(R.id.rbSalad)
        val tvSaved = findViewById<TextView>(R.id.tvSaved)
        val cbMeonis = findViewById<CheckBox>(R.id.cbMeonis)
        val cbWofels = findViewById<CheckBox>(R.id.cbWofels)
          val btSubmit = findViewById<Button>(R.id.btSubmit)

        btSubmit.setOnClickListener(){
            val rbButtonStatus = rgGroup.checkedRadioButtonId
            val meet = findViewById<RadioButton>(rbButtonStatus)
            val meonis = cbMeonis.isChecked
            val wofels = cbWofels.isChecked

            val output = "You orders Burger with : \n"+
                    "${meet.text}\n"+
                    (if(meonis) "Meonis \n" else "")+
                    (if(wofels) "Wofels \n" else "")

            tvSaved.text = output
        }
    }
}